app/src/main/java/org/p2p/wallet/auth/ui/smsinput/NewSmsInputPresenter.kt

```diff
@@ -104,7 +104,7 @@ class NewSmsInputPresenter(
 
             restoreWalletRestoreInteractor.finishRestoreCustomShare(smsCode)
 
-            if (userRestoreInteractor.isUserReadyToBeRestored()) {
+            if (userRestoreInteractor.isUserReadyToBeRestoredByPhone()) {
```

Comment:
you can pass RestoreWay into the method and return result depending on it

userRestoreInteractor.isUserReadyToBeRestored(SocialPlusCustomShare) for example